docs/GettingStartedTorch.md

```diff
@@ -106,6 +106,7 @@ labelHook             | function     | No        | A function(input,dblabel) tha
 trainBatchSize        | number       | No        | If specified, sets train batch size. May be overridden by user in DIGITS UI.
 validationBatchSize   | number       | No        | If specified, sets validation batch size. May be overridden by user in DIGITS UI.
 fineTuneHook          | function     | No        | A function(net) that returns the model to be used for fine-tuning. The untuned model is passed as a function parameter.
+disableAutoDataParallelism | boolean | No        | By default models are encapsulated in a nn.DataParallelTable container to enable multi-GPU training when more than 1 GPUs are selected. Setting this flag to `true` disables this mechanism.
```

Comment:
Actually I'm a bit unsure... with `disableAutoDataParallelism` the user can disable encapsulation with `disableAutoDataParallelism=true` and the Lua code looks like:

```lua
if not network.disableAutoDataParallelism then
```

If I change the flag to `enableAutoDataParallelism` then the user can disable automatic encapsulation with  `enableAutoDataParallelism=false` and the Lua code looks like:

```lua
if network.enableAutoDataParallelism ~= false then
```

The former method sounds more like positive logic to me given that we want to give an option to disable encapsulation (as we want encapsulation by default).